src/crypto_utils.ts

```diff
@@ -248,7 +275,7 @@ export function decodeUint(plaintextBytes: Uint8Array): bigint {
     return BigInt("0x" + plaintext.join(""))
 }
 
-function encryptRandomNumber(r: string | Uint8Array, key: Uint8Array) {
+export function encryptNumber(r: string | Uint8Array, key: Uint8Array) {
```

Comment:
you encrypt a string, or a UInt8Array . is the function name correct?

Reply:
In both cases it is a random number that is being encrypted, it can be one of two different types